.circleci/config.yml

```diff
@@ -299,7 +299,7 @@ jobs:
             pip install --user codespell
       - run:
           name: Check spelling
-          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*" -I ./scripts/codespell_whitelist.txt
+          command: ~/.local/bin/codespell --check-filenames -S "*.enc,.git,Dockerfile*,_*" -I ./scripts/codespell_whitelist.txt
```

Comment:
Which files is this meant to exclude? I don't see any files starting with `_` in the PR.

On the other hand we have `docs/_templates/` and `docs/_static/`. They currently do not contain too much text but still, I don't see a reason to exclude them.

Reply:
The main problem here was that the OZ sources contained a lot of spelling errors and/or unknown words. So I thought it made sense to exclude external sources from being spell-checked.